apps/passport-client/components/screens/SecondPartyTicketVerifyScreen.tsx

```diff
@@ -126,7 +129,8 @@ export function SecondPartyTicketVerifyScreen() {
   let connectionError = false;
 
   const checkAndVerifyComplete =
-    verifyResult !== undefined && (!pcd || checkResult !== undefined);
+    verifyResult !== undefined &&
+    (!pcd || !isDevconnectTicket(pcd) || checkResult !== undefined);
```

Comment:
having a hard time parsing the intention here

Reply:
I've added a comment explaining the logic